docs/source/user-guide/python.md

```diff
@@ -28,9 +28,15 @@ popular file formats files, run it in a distributed environment, and obtain the
 
 The following code demonstrates how to create a Ballista context and connect to a scheduler.
 
+If you are running a standalone cluster (runs locally), all you need to do is call the stand alone cluster method `standalone()` or your BallistaContext. If you are running a cluster in remote mode, you need to provide the URL `BallistaContext.remote("http://my-remote-ip:50050")`.
+
 ```text
->>> import ballista
->>> ctx = ballista.BallistaContext("localhost", 50050)
+>>> from pyballista import StandaloneBallista, RemoteBallista
```

Comment:
should `from pyballista` be reverted back to original name `ballista`? , there is another few imports like this

Reply:
Yes , I will update the docs